networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc

```diff
@@ -1,27 +1,33 @@
 :_content-type: ASSEMBLY
-[id="configuring-ingress-cluster-traffic-aws-network-load-balancer"]
-= Configuring ingress cluster traffic on AWS using a Network Load Balancer
+[id="configuring-ingress-cluster-traffic-aws"]
+= Configuring ingress cluster traffic on AWS
 include::_attributes/common-attributes.adoc[]
-:context: configuring-ingress-cluster-traffic-aws-network-load-balancer
+:context: configuring-ingress-cluster-traffic-aws
 
 toc::[]
 
-{product-title} provides methods for communicating from outside the cluster with services running in the cluster. This method uses a Network Load Balancer (NLB), which forwards the client's IP address to the node. You can configure an NLB on a new or existing AWS cluster.
+{product-title} provides methods for communicating from outside the cluster with services running in the cluster. This method uses load balancers on AWS. You can configure these load balancers on a new or existing AWS cluster.
 
-include::modules/nw-aws-replacing-clb-with-nlb.adoc[leveloffset=+1]
+include::modules/nw-configuring-elb-timeouts-aws-classic.adoc[leveloffset=+1]
 
-include::modules/nw-aws-nlb-existing-cluster.adoc[leveloffset=+1]
+include::modules/nw-configuring-route-timeouts.adoc[leveloffset=+2]
+
+include::modules/nw-configuring-clb-timeouts.adoc[leveloffset=+2]
+
+include::modules/nw-configuring-ingress-cluster-traffic-aws-networking-load-balancer.adoc[leveloffset=+1]
+
+include::modules/nw-aws-replacing-clb-with-nlb.adoc[leveloffset=+2]
+
+include::modules/nw-aws-nlb-existing-cluster.adoc[leveloffset=+2]
 
 [IMPORTANT]
 ====
 Before you can configure an Ingress Controller NLB on a new AWS cluster, you must complete the xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installation-initializing_installing-aws-network-customizations[Creating the installation configuration file] procedure.
 ====
 
-include::modules/nw-aws-nlb-new-cluster.adoc[leveloffset=+1]
+include::modules/nw-aws-nlb-new-cluster.adoc[leveloffset=+2]
 
-[role="_additional-resources"]
-[id="configuring-ingress-cluster-traffic-aws-network-load-balancer-addtl-resources"]
 == Additional resources
 
 * xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[Installing a cluster on AWS with network customizations].
-* For more information, see link:https://kubernetes.io/docs/concepts/services-networking/service/#aws-nlb-support[Network Load Balancer support on AWS].
+* For more information, see link:https://kubernetes.io/docs/concept/services-networking/service/#aws-nlb-support[Network Load Balancer support on AWS].
```

Comment:
The link with "concepts" works for me, and the link with "concept" does not.  Was this an accidental search-and-replace?

Reply:
Great catch, thank you! Fixed